gradle/versions.gradle

```diff
@@ -159,7 +159,7 @@ dependencyManagement {
     dependency 'org.openjdk.jmh:jmh-core:1.34'
     dependency 'org.openjdk.jmh:jmh-generator-annprocess:1.34'
 
-    dependency 'org.rocksdb:rocksdbjni:6.15.2'
+    dependency 'org.rocksdb:rocksdbjni:6.29.4'
```

Comment:
At the start of all fast-sync's we clear at a minimum the worldstate.  The problem seems to be that when we call `RocksDbKeyValueStorage.clear(segmentHandle)`, we call 
```
deleteRange(final ColumnFamilyHandle columnFamilyHandle,
      final byte[] beginKey, final byte[] endKey)
```

which is a defaulted overload for:
```
 RocksDB.deleteRange(private native void deleteRange(final long handle, final byte[] beginKey,
      final int beginKeyOffset, final int beginKeyLength, final byte[] endKey,
      final int endKeyOffset, final int endKeyLength, final long cfHandle)
```

which is throwing `NotSupported`.  I am digging around in the native codebase to find out if it is indeed no longer supported and if so what the alternative is.  

It is unfortunate that the jni interface offers up a deprecated signature with no warning or annotation... 🤔

Reply:
I am not sure why deleteRange is throwing NotSupported, but on a local branch I have made some changes to `RocksDBColumnarKeyValueStorage.clear()` to just drop and recreate the column family rather than deleting each and every key via iterator.  I am testing this locally and can push a suggested change to this PR.

Anecdotally, I think the drop/recreate strategy might resolve #3682 which is experiencing a long blocking behavior on this call.  🤞

Reply:
this can be resolved by dropping the column family and recreating it instead of iterating over all of the keys in the segmenthandle.  This should resolve the blocking `clear()` operations on #3682 also.

Reply:
I don't see how deleting the column will physically delete the data. and if we have to iterate on all the data of the segment, those can be very long or even block Besu. Have you had the opportunity to test this technique on a database filled

Reply:
The last time I had tried this technique it had freeze all Besu because of the number of keys

Reply:
Are we sure that after everything will be removed ?

Reply:
anecdotally I see the account db storage getting reclaimed on a restart, hopefully I will catch a re-pivot and see it happening organically.  I will confirm/disconfirm tomorrow.